Pin Python 3.12 and update compiler stack for Sequoia 15.4, update libMesh

milljm opened the pull request:
Initial commit attempting to support Python 3.12. This also updates the conda compiler stack (clang 16 --> clang 18 on MacOS) in order to support Sequoia 15.4.

It was found a libMesh update is also necessary to solve recent Sequoia 15.4.x update.

Closes #29764
Closes #30227
Closes #30349
Closes #29830

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2660044/), step Conda build config check on 77df3ba wanted to post the following:

A change of the following file(s) triggered this check:

  conda/peacock

The following file(s) are unchanged:

  conda/mpi/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2660044/), step Conda PETSc build config notification on 77df3ba wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2660044/), step Conda libmesh build config notification on 77df3ba wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2660374/), step Conda build config check on e7e7eb7 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/peacock

The following file(s) are unchanged:

  conda/mpi/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2660374/), step Conda libmesh build config notification on e7e7eb7 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/2660405/), step Docs: sync website on e7e7eb7 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/29765/site)

This comment will be updated on new commits.

moosebuild commented:
Job [Coverage](https://civet.inl.gov/job/2660421/), step Generate coverage on e7e7eb7 wanted to post the following:


## Framework coverage

<table>
  <thead>
    <tr>
      <th></th>
      <th>4d0bbd</th>
      <th colspan=4>#29765 e7e7eb</th>
    </tr>
    <tr>
      <th></th>
      <th>Total</th>
      <th>Total</th>
      <th>+/-</th>
      <th>New</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><b>Rate</b></td>
      <td>85.24%</td>
      <td>85.24%</td>
      <td>-0.00%</td>
      <td>-</td>
    </tr>
    <tr>
      <td><b>Hits</b></td>
      <td>108456</td>
      <td>108455</td>
      <td>-1</td>
      <td>0</td>
    </tr>
    <tr>
      <td><b>Misses</b></td>
      <td>18781</td>
      <td>18782</td>
      <td>+1</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

[Diff coverage report](https://mooseframework.inl.gov/docs/PRs/29765/coverage_diff/framework)

[Full coverage report](https://mooseframework.inl.gov/docs/PRs/29765/coverage/framework)


## Modules coverage

Coverage did not change

## Full coverage reports

<details>
<summary>Reports</summary>

- [`framework`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/framework)
- [`chemical_reactions`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/chemical_reactions)
- [`combined`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/combined)
- [`contact`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/contact)
- [`electromagnetics`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/electromagnetics)
- [`external_petsc_solver`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/external_petsc_solver)
- [`fluid_properties`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/fluid_properties)
- [`fsi`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/fsi)
- [`functional_expansion_tools`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/functional_expansion_tools)
- [`geochemistry`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/geochemistry)
- [`heat_transfer`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/heat_transfer)
- [`level_set`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/level_set)
- [`misc`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/misc)
- [`navier_stokes`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/navier_stokes)
- [`optimization`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/optimization)
- [`peridynamics`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/peridynamics)
- [`phase_field`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/phase_field)
- [`porous_flow`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/porous_flow)
- [`ray_tracing`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/ray_tracing)
- [`rdg`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/rdg)
- [`reactor`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/reactor)
- [`richards`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/richards)
- [`scalar_transport`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/scalar_transport)
- [`solid_mechanics`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/solid_mechanics)
- [`solid_properties`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/solid_properties)
- [`stochastic_tools`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/stochastic_tools)
- [`thermal_hydraulics`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/thermal_hydraulics)
- [`xfem`](https://mooseframework.inl.gov/docs/PRs/29765/coverage/xfem)

</details>



This comment will be updated on new commits.

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2749198/), step Versioner verify on f5c0ebf wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 8 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 5abb5f2 | 9269e0c   | 2024.12.23                      | 2025.03.14                      |
| mpi         | CHANGE   | 82bbd28 | 3899907   | 2024.12.23                      | 2025.03.14                      |
| wasp        | CHANGE   | 46ea1c0 | 29a13b8   | 2025.02.25 build 0              | 2025.02.25 build 1              |
| petsc       | CHANGE   | e78a9a4 | 9bb2eba   | 3.22.1.193.g72c1e49ee3d build 1 | 3.22.1.193.g72c1e49ee3d build 2 |
| libmesh-vtk | CHANGE   | 9a4dbab | 44e467e   | 9.3.0 build 5                   | 9.3.0 build 6                   |
| peacock     | CHANGE   | ef44cb6 | 68d528e   | 2024.12.23                      | 2025.03.14                      |
| libmesh     | CHANGE   | dabc922 | f127336   | 2025.02.25 build 0              | 2025.02.25 build 1              |
| moose-dev   | CHANGE   | fcaa73c | bafb09b   | 2025.03.03                      | 2025.03.14                      |

Commit:
Bump Python to 3.12

Begin supporting Python 3.12.
Prefering class imports over module imports when locating python class syntax
Add CMake constraints (>3<4)

Refs #30213
Closes #29764

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2775793/), step Versioner verify on a96cefa wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 8 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 8af34cd | bd14d2b   | 2025.03.24                      | 2025.03.25                      |
| mpi         | CHANGE   | d5b4334 | e1e0e0e   | 2025.03.24                      | 2025.03.25                      |
| wasp        | CHANGE   | a8340b3 | 31ed482   | 2025.02.25 build 1              | 2025.02.25 build 2              |
| libmesh-vtk | CHANGE   | 83f29bc | 0b064f5   | 9.3.0 build 6                   | 9.3.0 build 7                   |
| peacock     | CHANGE   | f8f84d6 | a0c3c31   | 2025.03.24                      | 2025.03.25                      |
| petsc       | CHANGE   | caeec6b | 89b8839   | 3.22.1.193.g72c1e49ee3d build 2 | 3.22.1.193.g72c1e49ee3d build 3 |
| libmesh     | CHANGE   | bc5c687 | b1f82ca   | 2025.03.06 build 1              | 2025.03.06 build 2              |
| moose-dev   | CHANGE   | 00b281f | dd5148d   | 2025.03.24                      | 2025.03.25                      |

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2776571/), step Versioner verify on f752ab5 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 8 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 8af34cd | bd14d2b   | 2025.03.24                      | 2025.03.25                      |
| mpi         | CHANGE   | d5b4334 | e1e0e0e   | 2025.03.24                      | 2025.03.25                      |
| wasp        | CHANGE   | a8340b3 | 31ed482   | 2025.02.25 build 1              | 2025.02.25 build 2              |
| libmesh-vtk | CHANGE   | 83f29bc | 0b064f5   | 9.3.0 build 6                   | 9.3.0 build 7                   |
| peacock     | CHANGE   | f8f84d6 | a0c3c31   | 2025.03.24                      | 2025.03.25                      |
| petsc       | CHANGE   | caeec6b | 89b8839   | 3.22.1.193.g72c1e49ee3d build 2 | 3.22.1.193.g72c1e49ee3d build 3 |
| libmesh     | CHANGE   | bc5c687 | b1f82ca   | 2025.03.06 build 1              | 2025.03.06 build 2              |
| moose-dev   | CHANGE   | 00b281f | dd5148d   | 2025.03.24                      | 2025.03.25                      |

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2783194/), step Versioner verify on 7beea35 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 8 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 8af34cd | bd14d2b   | 2025.03.24                      | 2025.03.25                      |
| mpi         | CHANGE   | d5b4334 | e1e0e0e   | 2025.03.24                      | 2025.03.25                      |
| wasp        | CHANGE   | a8340b3 | 31ed482   | 2025.02.25 build 1              | 2025.02.25 build 2              |
| libmesh-vtk | CHANGE   | 83f29bc | 0b064f5   | 9.3.0 build 6                   | 9.3.0 build 7                   |
| peacock     | CHANGE   | f8f84d6 | a0c3c31   | 2025.03.24                      | 2025.03.25                      |
| petsc       | CHANGE   | caeec6b | 89b8839   | 3.22.1.193.g72c1e49ee3d build 2 | 3.22.1.193.g72c1e49ee3d build 3 |
| libmesh     | CHANGE   | bc5c687 | b1f82ca   | 2025.03.06 build 1              | 2025.03.06 build 2              |
| moose-dev   | CHANGE   | 00b281f | dd5148d   | 2025.03.24                      | 2025.03.25                      |

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2784165/), step Versioner verify on d422c32 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 8 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 8af34cd | 8cd33bc   | 2025.03.24                      | 2025.03.28                      |
| mpi         | CHANGE   | d5b4334 | a07c401   | 2025.03.24                      | 2025.03.28                      |
| wasp        | CHANGE   | a8340b3 | 31ed482   | 2025.02.25 build 1              | 2025.02.25 build 2              |
| libmesh-vtk | CHANGE   | 83f29bc | a290063   | 9.3.0 build 6                   | 9.3.0 build 7                   |
| peacock     | CHANGE   | f8f84d6 | 0243cd4   | 2025.03.24                      | 2025.03.28                      |
| petsc       | CHANGE   | caeec6b | d6c69b9   | 3.22.1.193.g72c1e49ee3d build 2 | 3.22.1.193.g72c1e49ee3d build 3 |
| libmesh     | CHANGE   | bc5c687 | 6bba889   | 2025.03.06 build 1              | 2025.03.06 build 2              |
| moose-dev   | CHANGE   | 00b281f | 859ff08   | 2025.03.24                      | 2025.03.28                      |

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2784485/), step Versioner verify on f2bb7e0 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 9 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 8af34cd | 8cd33bc   | 2025.03.24                      | 2025.03.28                      |
| mpi         | CHANGE   | d5b4334 | 77c94ca   | 2025.03.24                      | 2025.03.28                      |
| wasp        | CHANGE   | a8340b3 | d94c0fa   | 2025.02.25 build 1              | 2025.02.25 build 2              |
| seacas      | CHANGE   | 067e220 | 4453051   | 2025.02.27 build 0              | 2025.02.27 build 1              |
| libmesh-vtk | CHANGE   | 83f29bc | 10a945c   | 9.3.0 build 6                   | 9.3.0 build 7                   |
| peacock     | CHANGE   | f8f84d6 | ad42478   | 2025.03.24                      | 2025.03.28                      |
| petsc       | CHANGE   | caeec6b | 6d7c71d   | 3.22.1.193.g72c1e49ee3d build 2 | 3.22.1.193.g72c1e49ee3d build 3 |
| libmesh     | CHANGE   | bc5c687 | a88a393   | 2025.03.06 build 1              | 2025.03.06 build 2              |
| moose-dev   | CHANGE   | 00b281f | 5e9fd15   | 2025.03.24                      | 2025.03.28                      |

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2784747/), step Versioner verify on 918520a wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 9 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 8af34cd | 8cd33bc   | 2025.03.24                      | 2025.03.28                      |
| mpi         | CHANGE   | d5b4334 | 77c94ca   | 2025.03.24                      | 2025.03.28                      |
| wasp        | CHANGE   | a8340b3 | d94c0fa   | 2025.02.25 build 1              | 2025.02.25 build 2              |
| seacas      | CHANGE   | 067e220 | 4453051   | 2025.02.27 build 0              | 2025.02.27 build 1              |
| libmesh-vtk | CHANGE   | 83f29bc | 10a945c   | 9.3.0 build 6                   | 9.3.0 build 7                   |
| peacock     | CHANGE   | f8f84d6 | ad42478   | 2025.03.24                      | 2025.03.28                      |
| petsc       | CHANGE   | caeec6b | 6d7c71d   | 3.22.1.193.g72c1e49ee3d build 2 | 3.22.1.193.g72c1e49ee3d build 3 |
| libmesh     | CHANGE   | bc5c687 | a88a393   | 2025.03.06 build 1              | 2025.03.06 build 2              |
| moose-dev   | CHANGE   | 00b281f | 5e9fd15   | 2025.03.24                      | 2025.03.28                      |

Commit:
Bump Apptaner moose-dev Python

Bump our Apptainer containers to use Python 3.12
Add bless.py script

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2787710/), step Versioner verify on 21c0564 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 9 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 8af34cd | 79f87a1   | 2025.03.24                      | 2025.03.31                      |
| mpi         | CHANGE   | d5b4334 | da85cac   | 2025.03.24                      | 2025.03.31                      |
| wasp        | CHANGE   | a8340b3 | d94c0fa   | 2025.02.25 build 1              | 2025.02.25 build 2              |
| seacas      | CHANGE   | 067e220 | 4453051   | 2025.02.27 build 0              | 2025.02.27 build 1              |
| libmesh-vtk | CHANGE   | 83f29bc | a288d69   | 9.3.0 build 6                   | 9.3.0 build 7                   |
| peacock     | CHANGE   | f8f84d6 | c4f7409   | 2025.03.24                      | 2025.03.31                      |
| petsc       | CHANGE   | caeec6b | c884ea2   | 3.22.1.193.g72c1e49ee3d build 2 | 3.22.1.193.g72c1e49ee3d build 3 |
| libmesh     | CHANGE   | bc5c687 | 4dee6d3   | 2025.03.06 build 1              | 2025.03.06 build 2              |
| moose-dev   | CHANGE   | 00b281f | 5ca64ab   | 2025.03.24                      | 2025.03.31                      |

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2794261/), step Versioner verify on deb9b22 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 9 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 8af34cd | 79f87a1   | 2025.03.24                      | 2025.03.31                      |
| mpi         | CHANGE   | d5b4334 | 27b8f95   | 2025.03.24                      | 2025.03.31                      |
| wasp        | CHANGE   | a8340b3 | a88e604   | 2025.02.25 build 1              | 2025.02.25 build 2              |
| seacas      | CHANGE   | 067e220 | d4ce403   | 2025.02.27 build 0              | 2025.02.27 build 1              |
| libmesh-vtk | CHANGE   | 83f29bc | 2850334   | 9.3.0 build 6                   | 9.3.0 build 7                   |
| peacock     | CHANGE   | f8f84d6 | 30d9a74   | 2025.03.24                      | 2025.03.31                      |
| petsc       | CHANGE   | caeec6b | bdac9b3   | 3.22.1.193.g72c1e49ee3d build 2 | 3.22.1.193.g72c1e49ee3d build 3 |
| libmesh     | CHANGE   | bc5c687 | 65ef7e8   | 2025.03.06 build 1              | 2025.03.06 build 2              |
| moose-dev   | CHANGE   | 00b281f | c664304   | 2025.03.24                      | 2025.03.31                      |

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2795362/), step Versioner verify on f1733d9 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 9 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 8af34cd | 79f87a1   | 2025.03.24                      | 2025.03.31                      |
| mpi         | CHANGE   | d5b4334 | e81f682   | 2025.03.24                      | 2025.03.31                      |
| wasp        | CHANGE   | a8340b3 | a88e604   | 2025.02.25 build 1              | 2025.02.25 build 2              |
| seacas      | CHANGE   | 067e220 | d4ce403   | 2025.02.27 build 0              | 2025.02.27 build 1              |
| libmesh-vtk | CHANGE   | 83f29bc | e68e5e3   | 9.3.0 build 6                   | 9.3.0 build 7                   |
| peacock     | CHANGE   | f8f84d6 | 3dd88d9   | 2025.03.24                      | 2025.03.31                      |
| petsc       | CHANGE   | caeec6b | 9dd9703   | 3.22.1.193.g72c1e49ee3d build 2 | 3.22.1.193.g72c1e49ee3d build 3 |
| libmesh     | CHANGE   | bc5c687 | b4f1916   | 2025.03.06 build 1              | 2025.03.06 build 2              |
| moose-dev   | CHANGE   | 00b281f | 7edc387   | 2025.03.24                      | 2025.03.31                      |

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2795474/), step Versioner verify on 5e5f76f wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 9 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 8af34cd | 79f87a1   | 2025.03.24                      | 2025.03.31                      |
| mpi         | CHANGE   | d5b4334 | 30a79af   | 2025.03.24                      | 2025.03.31                      |
| wasp        | CHANGE   | a8340b3 | a88e604   | 2025.02.25 build 1              | 2025.02.25 build 2              |
| seacas      | CHANGE   | 067e220 | d4ce403   | 2025.02.27 build 0              | 2025.02.27 build 1              |
| libmesh-vtk | CHANGE   | 83f29bc | 62b3e76   | 9.3.0 build 6                   | 9.3.0 build 7                   |
| peacock     | CHANGE   | f8f84d6 | e205f36   | 2025.03.24                      | 2025.03.31                      |
| petsc       | CHANGE   | caeec6b | 9a014a3   | 3.22.1.193.g72c1e49ee3d build 2 | 3.22.1.193.g72c1e49ee3d build 3 |
| libmesh     | CHANGE   | bc5c687 | 8fb8f48   | 2025.03.06 build 1              | 2025.03.06 build 2              |
| moose-dev   | CHANGE   | 00b281f | 10e6ea4   | 2025.03.24                      | 2025.03.31                      |

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2796986/), step Versioner verify on cb5a48f wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 9 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 8af34cd | 79f87a1   | 2025.03.24                      | 2025.03.31                      |
| mpi         | CHANGE   | d5b4334 | f3a4714   | 2025.03.24                      | 2025.03.31                      |
| wasp        | CHANGE   | a8340b3 | a88e604   | 2025.02.25 build 1              | 2025.02.25 build 2              |
| seacas      | CHANGE   | 067e220 | d4ce403   | 2025.02.27 build 0              | 2025.02.27 build 1              |
| libmesh-vtk | CHANGE   | 83f29bc | e888fb1   | 9.3.0 build 6                   | 9.3.0 build 7                   |
| peacock     | CHANGE   | f8f84d6 | 1ee9095   | 2025.03.24                      | 2025.03.31                      |
| petsc       | CHANGE   | caeec6b | 121e95a   | 3.22.1.193.g72c1e49ee3d build 2 | 3.22.1.193.g72c1e49ee3d build 3 |
| libmesh     | CHANGE   | bc5c687 | 8cc0f77   | 2025.03.06 build 1              | 2025.03.06 build 2              |
| moose-dev   | CHANGE   | 00b281f | 3e815fb   | 2025.03.24                      | 2025.03.31                      |

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2797101/), step Versioner verify on bd35aae wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 9 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 8af34cd | 79f87a1   | 2025.03.24                      | 2025.03.31                      |
| mpi         | CHANGE   | d5b4334 | ff5da75   | 2025.03.24                      | 2025.03.31                      |
| wasp        | CHANGE   | a8340b3 | a88e604   | 2025.02.25 build 1              | 2025.02.25 build 2              |
| seacas      | CHANGE   | 067e220 | d4ce403   | 2025.02.27 build 0              | 2025.02.27 build 1              |
| libmesh-vtk | CHANGE   | 83f29bc | 002cc36   | 9.3.0 build 6                   | 9.3.0 build 7                   |
| peacock     | CHANGE   | f8f84d6 | 6885e63   | 2025.03.24                      | 2025.03.31                      |
| petsc       | CHANGE   | caeec6b | 7daca87   | 3.22.1.193.g72c1e49ee3d build 2 | 3.22.1.193.g72c1e49ee3d build 3 |
| libmesh     | CHANGE   | bc5c687 | d7a26a0   | 2025.03.06 build 1              | 2025.03.06 build 2              |
| moose-dev   | CHANGE   | 00b281f | 17fc41a   | 2025.03.24                      | 2025.03.31                      |

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2801004/), step Versioner verify on fb0d8f0 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 9 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 8af34cd | 79f87a1   | 2025.03.24                      | 2025.03.31                      |
| mpi         | CHANGE   | d5b4334 | e55893f   | 2025.03.24                      | 2025.03.31                      |
| wasp        | CHANGE   | a8340b3 | a88e604   | 2025.02.25 build 1              | 2025.02.25 build 2              |
| seacas      | CHANGE   | 067e220 | d4ce403   | 2025.02.27 build 0              | 2025.02.27 build 1              |
| libmesh-vtk | CHANGE   | 83f29bc | 69d7750   | 9.3.0 build 6                   | 9.3.0 build 7                   |
| peacock     | CHANGE   | f8f84d6 | 9a1033e   | 2025.03.24                      | 2025.03.31                      |
| petsc       | CHANGE   | caeec6b | 2ef7a78   | 3.22.1.193.g72c1e49ee3d build 2 | 3.22.1.193.g72c1e49ee3d build 3 |
| libmesh     | CHANGE   | bc5c687 | 13ddf34   | 2025.03.06 build 1              | 2025.03.06 build 2              |
| moose-dev   | CHANGE   | 00b281f | 88cb264   | 2025.03.24                      | 2025.03.31                      |

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2802064/), step Versioner verify on 9598a1f wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 9 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 8af34cd | 79f87a1   | 2025.03.24                      | 2025.03.31                      |
| mpi         | CHANGE   | d5b4334 | e55893f   | 2025.03.24                      | 2025.03.31                      |
| wasp        | CHANGE   | a8340b3 | a88e604   | 2025.02.25 build 1              | 2025.02.25 build 2              |
| seacas      | CHANGE   | 067e220 | d4ce403   | 2025.02.27 build 0              | 2025.02.27 build 1              |
| libmesh-vtk | CHANGE   | 83f29bc | 69d7750   | 9.3.0 build 6                   | 9.3.0 build 7                   |
| peacock     | CHANGE   | f8f84d6 | 0842293   | 2025.03.24                      | 2025.03.31                      |
| petsc       | CHANGE   | caeec6b | 2ef7a78   | 3.22.1.193.g72c1e49ee3d build 2 | 3.22.1.193.g72c1e49ee3d build 3 |
| libmesh     | CHANGE   | bc5c687 | 13ddf34   | 2025.03.06 build 1              | 2025.03.06 build 2              |
| moose-dev   | CHANGE   | 00b281f | a3a8fc1   | 2025.03.24                      | 2025.03.31                      |

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2805118/), step Versioner verify on 578ec0e wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 9 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 8af34cd | 715d18d   | 2025.03.24                      | 2025.04.08                      |
| mpi         | CHANGE   | d5b4334 | 6d5c684   | 2025.03.24                      | 2025.04.08                      |
| wasp        | CHANGE   | a8340b3 | 5dc9e45   | 2025.02.25 build 1              | 2025.02.25 build 2              |
| seacas      | CHANGE   | 067e220 | d1c59f8   | 2025.02.27 build 0              | 2025.02.27 build 1              |
| libmesh-vtk | CHANGE   | 83f29bc | 5632c68   | 9.3.0 build 6                   | 9.3.0 build 7                   |
| peacock     | CHANGE   | f8f84d6 | 00c35a4   | 2025.03.24                      | 2025.04.08                      |
| petsc       | CHANGE   | caeec6b | 53615ca   | 3.22.1.193.g72c1e49ee3d build 2 | 3.22.1.193.g72c1e49ee3d build 3 |
| libmesh     | CHANGE   | bc5c687 | 5e73fd7   | 2025.03.06 build 1              | 2025.03.06 build 2              |
| moose-dev   | CHANGE   | 00b281f | 99c7d31   | 2025.03.24                      | 2025.04.08                      |

Commit:
Bump to Clang 18 on Darwin

Bump Apple Si MacOS SDK requirements.
Adding Sequoia patch for moose-seacas.

Closes #29764 #30227

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2807848/), step Versioner verify on ec22283 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 10 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 8af34cd | 715d18d   | 2025.03.24                      | 2025.04.08                      |
| mpi         | CHANGE   | d5b4334 | ef81563   | 2025.03.24                      | 2025.04.08                      |
| wasp        | CHANGE   | a8340b3 | e181ec5   | 2025.02.25 build 1              | 2025.02.25 build 2              |
| pprof       | CHANGE   | fe051bc | 1acc807   | 2024.12.23 build 0              | 2025.04.08 build 0              |
| seacas      | CHANGE   | 067e220 | 5d0701b   | 2025.02.27 build 0              | 2025.02.27 build 1              |
| libmesh-vtk | CHANGE   | 83f29bc | 11ee918   | 9.3.0 build 6                   | 9.4.2 build 0                   |
| peacock     | CHANGE   | f8f84d6 | b0ccd7b   | 2025.03.24                      | 2025.04.08                      |
| petsc       | CHANGE   | caeec6b | d98e4aa   | 3.22.1.193.g72c1e49ee3d build 2 | 3.22.1.193.g72c1e49ee3d build 3 |
| libmesh     | CHANGE   | bc5c687 | a1e9a7b   | 2025.03.06 build 1              | 2025.03.06 build 2              |
| moose-dev   | CHANGE   | 00b281f | d13b8ff   | 2025.03.24                      | 2025.04.08                      |

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2808048/), step Versioner verify on 1e08a58 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 10 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 8af34cd | e29409b   | 2025.03.24                      | 2025.04.08                      |
| mpi         | CHANGE   | d5b4334 | ef81563   | 2025.03.24                      | 2025.04.08                      |
| wasp        | CHANGE   | a8340b3 | e181ec5   | 2025.02.25 build 1              | 2025.02.25 build 2              |
| pprof       | CHANGE   | fe051bc | 1acc807   | 2024.12.23 build 0              | 2025.04.08 build 0              |
| seacas      | CHANGE   | 067e220 | 5d0701b   | 2025.02.27 build 0              | 2025.02.27 build 1              |
| libmesh-vtk | CHANGE   | 83f29bc | 11ee918   | 9.3.0 build 6                   | 9.4.2 build 0                   |
| peacock     | CHANGE   | f8f84d6 | b0ccd7b   | 2025.03.24                      | 2025.04.08                      |
| petsc       | CHANGE   | caeec6b | d98e4aa   | 3.22.1.193.g72c1e49ee3d build 2 | 3.22.1.193.g72c1e49ee3d build 3 |
| libmesh     | CHANGE   | bc5c687 | a1e9a7b   | 2025.03.06 build 1              | 2025.03.06 build 2              |
| moose-dev   | CHANGE   | 00b281f | 1dde603   | 2025.03.24                      | 2025.04.08                      |

Commit:
Move tirpc handling

Have the configure_libmesh.sh script instead, handle the libtirpc
path swap (so ./update_and_rebuild_libmesh.sh works).

Move libtirpc package install into moose-mpi (so above works).

Commit:
remove bless.py

I think we are solving this with a first-class solution.

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2812462/), step Versioner verify on a82aaa2 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 10 changed, 0 failed

| package     | status   | hash    | to hash   | version                         | to version                      |
|-------------|----------|---------|-----------|---------------------------------|---------------------------------|
| tools       | CHANGE   | 8af34cd | e29409b   | 2025.03.24                      | 2025.04.08                      |
| mpi         | CHANGE   | d5b4334 | bfe9689   | 2025.03.24                      | 2025.04.08                      |
| wasp        | CHANGE   | a8340b3 | e181ec5   | 2025.02.25 build 1              | 2025.02.25 build 2              |
| pprof       | CHANGE   | fe051bc | 1acc807   | 2024.12.23 build 0              | 2025.04.08 build 0              |
| seacas      | CHANGE   | 067e220 | 5d0701b   | 2025.02.27 build 0              | 2025.02.27 build 1              |
| libmesh-vtk | CHANGE   | 83f29bc | 962f09f   | 9.3.0 build 6                   | 9.4.2 build 0                   |
| peacock     | CHANGE   | f8f84d6 | 4d294e2   | 2025.03.24                      | 2025.04.08                      |
| petsc       | CHANGE   | caeec6b | a3ec3b4   | 3.22.1.193.g72c1e49ee3d build 2 | 3.22.1.193.g72c1e49ee3d build 3 |
| libmesh     | CHANGE   | bc5c687 | 29a1a20   | 2025.03.06 build 1              | 2025.03.06 build 2              |
| moose-dev   | CHANGE   | 00b281f | 954cc8c   | 2025.03.24                      | 2025.04.08                      |

Commit:
Rebase on Next

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2813060/), step Versioner verify on bfd21c0 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 10 changed, 0 failed

| package     | status   | hash    | to hash   | version                       | to version                    |
|-------------|----------|---------|-----------|-------------------------------|-------------------------------|
| tools       | CHANGE   | 8af34cd | 644d833   | 2025.03.24                    | 2025.04.11                    |
| mpi         | CHANGE   | d5b4334 | 3873729   | 2025.03.24                    | 2025.04.11                    |
| wasp        | CHANGE   | a8340b3 | e181ec5   | 2025.02.25 build 1            | 2025.02.25 build 2            |
| pprof       | CHANGE   | fe051bc | 2e4473d   | 2024.12.23 build 0            | 2025.04.11 build 0            |
| seacas      | CHANGE   | 067e220 | 5d0701b   | 2025.02.27 build 0            | 2025.02.27 build 1            |
| libmesh-vtk | CHANGE   | 83f29bc | c537cb7   | 9.3.0 build 6                 | 9.4.2 build 0                 |
| peacock     | CHANGE   | f8f84d6 | aca1cdf   | 2025.03.24                    | 2025.04.11                    |
| petsc       | CHANGE   | 2eee4ef | 0e984bc   | 3.23.0.6.gd9d7fd11dca build 0 | 3.23.0.6.gd9d7fd11dca build 1 |
| libmesh     | CHANGE   | d7c8d15 | 77038b0   | 2025.04.02 build 0            | 2025.04.02 build 1            |
| moose-dev   | CHANGE   | b61c771 | be49631   | 2025.04.02                    | 2025.04.11                    |

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2813161/), step Versioner verify on 478f313 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 10 changed, 0 failed

| package     | status   | hash    | to hash   | version                       | to version                    |
|-------------|----------|---------|-----------|-------------------------------|-------------------------------|
| tools       | CHANGE   | 8af34cd | 644d833   | 2025.03.24                    | 2025.04.11                    |
| mpi         | CHANGE   | d5b4334 | 3873729   | 2025.03.24                    | 2025.04.11                    |
| wasp        | CHANGE   | a8340b3 | e181ec5   | 2025.02.25 build 1            | 2025.02.25 build 2            |
| pprof       | CHANGE   | fe051bc | 2e4473d   | 2024.12.23 build 0            | 2025.04.11 build 0            |
| seacas      | CHANGE   | 067e220 | 5d0701b   | 2025.02.27 build 0            | 2025.02.27 build 1            |
| libmesh-vtk | CHANGE   | 83f29bc | c537cb7   | 9.3.0 build 6                 | 9.4.2 build 0                 |
| peacock     | CHANGE   | f8f84d6 | aca1cdf   | 2025.03.24                    | 2025.04.11                    |
| petsc       | CHANGE   | 2eee4ef | 0e984bc   | 3.23.0.6.gd9d7fd11dca build 0 | 3.23.0.6.gd9d7fd11dca build 1 |
| libmesh     | CHANGE   | d7c8d15 | dcf1c08   | 2025.04.02 build 0            | 2025.04.02 build 1            |
| moose-dev   | CHANGE   | b61c771 | 81c2bb2   | 2025.04.02                    | 2025.04.11                    |

Commit:
pin clang-tools

When we don't pin, we end up with an actual clang binary on linux
systems when installing the following combination:

conda create -n testing clang-format=18 clang-tools pycobertura

Which breaks building WASP in the moose-dev-clang container (because
the above installs an ancient clang v11)

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2815785/), step Versioner verify on 3a84236 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 10 changed, 0 failed

| package     | status   | hash    | to hash   | version                       | to version                    |
|-------------|----------|---------|-----------|-------------------------------|-------------------------------|
| tools       | CHANGE   | 8af34cd | 9265542   | 2025.03.24                    | 2025.04.11                    |
| mpi         | CHANGE   | d5b4334 | 3873729   | 2025.03.24                    | 2025.04.11                    |
| wasp        | CHANGE   | a8340b3 | e181ec5   | 2025.02.25 build 1            | 2025.02.25 build 2            |
| pprof       | CHANGE   | fe051bc | 2e4473d   | 2024.12.23 build 0            | 2025.04.11 build 0            |
| seacas      | CHANGE   | 067e220 | 5d0701b   | 2025.02.27 build 0            | 2025.02.27 build 1            |
| libmesh-vtk | CHANGE   | 83f29bc | c537cb7   | 9.3.0 build 6                 | 9.4.2 build 0                 |
| peacock     | CHANGE   | f8f84d6 | aca1cdf   | 2025.03.24                    | 2025.04.11                    |
| petsc       | CHANGE   | 2eee4ef | 0e984bc   | 3.23.0.6.gd9d7fd11dca build 0 | 3.23.0.6.gd9d7fd11dca build 1 |
| libmesh     | CHANGE   | d7c8d15 | dcf1c08   | 2025.04.02 build 0            | 2025.04.02 build 1            |
| moose-dev   | CHANGE   | b61c771 | 60b88e3   | 2025.04.02                    | 2025.04.11                    |

Commit:
rebase on next, bump version

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2820315/), step Versioner verify on 478f313 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 10 changed, 0 failed

| package     | status   | hash    | to hash   | version                       | to version                    |
|-------------|----------|---------|-----------|-------------------------------|-------------------------------|
| tools       | CHANGE   | 8af34cd | 644d833   | 2025.03.24                    | 2025.04.11                    |
| mpi         | CHANGE   | d5b4334 | 3873729   | 2025.03.24                    | 2025.04.11                    |
| wasp        | CHANGE   | a8340b3 | e181ec5   | 2025.02.25 build 1            | 2025.02.25 build 2            |
| pprof       | CHANGE   | fe051bc | 2e4473d   | 2024.12.23 build 0            | 2025.04.11 build 0            |
| seacas      | CHANGE   | 067e220 | 5d0701b   | 2025.02.27 build 0            | 2025.02.27 build 1            |
| libmesh-vtk | CHANGE   | 83f29bc | c537cb7   | 9.3.0 build 6                 | 9.4.2 build 0                 |
| peacock     | CHANGE   | f8f84d6 | aca1cdf   | 2025.03.24                    | 2025.04.11                    |
| petsc       | CHANGE   | 2eee4ef | 0e984bc   | 3.23.0.6.gd9d7fd11dca build 0 | 3.23.0.6.gd9d7fd11dca build 1 |
| libmesh     | CHANGE   | d7c8d15 | dcf1c08   | 2025.04.02 build 0            | 2025.04.02 build 1            |
| moose-dev   | CHANGE   | b61c771 | 81c2bb2   | 2025.04.02                    | 2025.04.11                    |

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2820441/), step Versioner verify on 62ced62 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 10 changed, 0 failed

| package     | status   | hash    | to hash   | version                       | to version                    |
|-------------|----------|---------|-----------|-------------------------------|-------------------------------|
| tools       | CHANGE   | 8af34cd | d66367f   | 2025.03.24                    | 2025.04.15                    |
| mpi         | CHANGE   | d5b4334 | 7c4af09   | 2025.03.24                    | 2025.04.15                    |
| wasp        | CHANGE   | a8340b3 | e181ec5   | 2025.02.25 build 1            | 2025.02.25 build 2            |
| pprof       | CHANGE   | fe051bc | 4507bcf   | 2024.12.23 build 0            | 2025.04.15 build 0            |
| seacas      | CHANGE   | 067e220 | 5d0701b   | 2025.02.27 build 0            | 2025.02.27 build 1            |
| libmesh-vtk | CHANGE   | 83f29bc | c48460e   | 9.3.0 build 6                 | 9.4.2 build 0                 |
| peacock     | CHANGE   | f8f84d6 | 33aeb84   | 2025.03.24                    | 2025.04.15                    |
| petsc       | CHANGE   | 2eee4ef | ecab4b7   | 3.23.0.6.gd9d7fd11dca build 0 | 3.23.0.6.gd9d7fd11dca build 1 |
| libmesh     | CHANGE   | 59aecc2 | 2030ad0   | 2025.04.02 build 1            | 2025.04.02 build 2            |
| moose-dev   | CHANGE   | 171cc5b | 1d93ea5   | 2025.04.11                    | 2025.04.15                    |

Commit:
including libMesh update

Add plotly to moose-tools
bump all dates (make them the same as libMesh update)

Closes #29830 #30349

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2823716/), step Versioner verify on b625c89 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 10 changed, 0 failed

| package     | status   | hash    | to hash   | version                       | to version                    |
|-------------|----------|---------|-----------|-------------------------------|-------------------------------|
| tools       | CHANGE   | 8af34cd | 827cc6f   | 2025.03.24                    | 2025.04.17                    |
| mpi         | CHANGE   | d5b4334 | a9161d1   | 2025.03.24                    | 2025.04.17                    |
| wasp        | CHANGE   | a8340b3 | e181ec5   | 2025.02.25 build 1            | 2025.02.25 build 2            |
| pprof       | CHANGE   | fe051bc | f8fd1e3   | 2024.12.23 build 0            | 2025.04.17 build 0            |
| seacas      | CHANGE   | 067e220 | 5d0701b   | 2025.02.27 build 0            | 2025.02.27 build 1            |
| libmesh-vtk | CHANGE   | 83f29bc | b994539   | 9.3.0 build 6                 | 9.4.2 build 0                 |
| peacock     | CHANGE   | f8f84d6 | a231b10   | 2025.03.24                    | 2025.04.17                    |
| petsc       | CHANGE   | 2eee4ef | ff9a7ec   | 3.23.0.6.gd9d7fd11dca build 0 | 3.23.0.6.gd9d7fd11dca build 1 |
| libmesh     | CHANGE   | 59aecc2 | cbc75b9   | 2025.04.02 build 1            | 2025.04.17 build 0            |
| moose-dev   | CHANGE   | 171cc5b | 1b20943   | 2025.04.11                    | 2025.04.17                    |

Commit:
bump hashes

moosebuild commented on libmesh:
```diff
@@ -1 +1 @@
-Subproject commit d524f884d1636756b12a7edee24673037509b51b
+Subproject commit b0ea29004a60fa0ce8a351c57f5f722f737ab297
```
**Caution!** This contains a submodule update

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2823880/), step Versioner verify on d64ae09 wanted to post the following:

### Versioner templates

Found 16 templates, 0 failed


### Versioner versions

Found 10 packages, 10 changed, 0 failed

| package     | status   | hash    | to hash   | version                       | to version                    |
|-------------|----------|---------|-----------|-------------------------------|-------------------------------|
| tools       | CHANGE   | 8af34cd | 827cc6f   | 2025.03.24                    | 2025.04.17                    |
| mpi         | CHANGE   | d5b4334 | a9161d1   | 2025.03.24                    | 2025.04.17                    |
| wasp        | CHANGE   | a8340b3 | e181ec5   | 2025.02.25 build 1            | 2025.02.25 build 2            |
| pprof       | CHANGE   | fe051bc | f8fd1e3   | 2024.12.23 build 0            | 2025.04.17 build 0            |
| seacas      | CHANGE   | 067e220 | 5d0701b   | 2025.02.27 build 0            | 2025.02.27 build 1            |
| libmesh-vtk | CHANGE   | 83f29bc | b994539   | 9.3.0 build 6                 | 9.4.2 build 0                 |
| peacock     | CHANGE   | f8f84d6 | a231b10   | 2025.03.24                    | 2025.04.17                    |
| petsc       | CHANGE   | 2eee4ef | ff9a7ec   | 3.23.0.6.gd9d7fd11dca build 0 | 3.23.0.6.gd9d7fd11dca build 1 |
| libmesh     | CHANGE   | 59aecc2 | cbc75b9   | 2025.04.02 build 1            | 2025.04.17 build 0            |
| moose-dev   | CHANGE   | 171cc5b | 1b20943   | 2025.04.11                    | 2025.04.17                    |

loganharbour commented:
HPC test is currently disabled due to HPC scratch outage. TMAP7 patch merged in https://github.com/idaholab/TMAP8/pull/257.
